tools/mk_deb_fpmdir.stack_blackbox.txt

```diff
@@ -0,0 +1,15 @@
+read /etc/profile.d/usrblackbox.sh               profile.d-usrblackbox.sh
```

Comment:
This file is the same as mk_rpm_fpmdir.stack_blackbox.txt.
I thought that Debian wants everything in /usr/bin, but that may just have been what a past co-worker told me.